tensorrt_llm/_torch/models/checkpoints/mistral/config_loader.py

```diff
@@ -301,7 +304,31 @@ def load(self, checkpoint_dir: str, **kwargs) -> ModelConfig:
             elif "FP8_BLOCK" in hf_quant_config.get("config_groups"):
                 quant_config.quant_algo = QuantAlgo.FP8_BLOCK_SCALES
                 quant_config.group_size = 128
-                quant_config.exclude_modules = ["*q_a_proj*", "*kv_a_proj_with_mqa*"]
+                quant_config.exclude_modules = [
+                    "language_model.model.layers.*.self_attn.q_a_proj*",
+                    "language_model.model.layers.*.self_attn.kv_a_proj_with_mqa*",
+                    "model.layers.*.self_attn.q_a_proj*",
+                    "model.layers.*.self_attn.kv_a_proj_with_mqa*",
+                ]
+        elif hf_quant_config.get("quant_method") == "fp8":
+            # Used for Eagle3 weight
+            if hf_quant_config.get("weight_block_size", []):
+                quant_config.quant_algo = QuantAlgo.FP8_BLOCK_SCALES
+                quant_config.exclude_modules = ["*kv_b_proj*", "*k_b_proj*", "*eh_proj"]
+
+                block_size = hf_quant_config.get("weight_block_size", [])
```

Comment:
The default value of `[]` does not seem required here either. The `assert` on the next line will stop the program here no matter if it's `[]` or `None`.

Reply:
But you might need to change the assert to `assert block_size is not None and tuple(block_size) == (128, 128)`. It's a bit more verbose but I think more explicit too.